language-server/modules/langserver-core/src/test/resources/completion/new/expression_context/config/query_expr_ctx_let_clause_config11.json

```diff
@@ -418,57 +418,57 @@
     {
       "label": "test",
       "kind": "Variable",
-      "detail": "$CompilationError$",
+      "detail": "int",
       "sortText": "B",
       "insertText": "test",
       "insertTextFormat": "Snippet"
     },
     {
-      "label": "c1",
+      "label": "personList",
       "kind": "Variable",
-      "detail": "$CompilationError$",
+      "detail": "$CompilationError$[]",
       "sortText": "B",
-      "insertText": "c1",
+      "insertText": "personList",
       "insertTextFormat": "Snippet"
     },
     {
-      "label": "c2",
+      "label": "person",
       "kind": "Variable",
       "detail": "$CompilationError$",
       "sortText": "B",
-      "insertText": "c2",
+      "insertText": "person",
       "insertTextFormat": "Snippet"
     },
     {
-      "label": "c3",
+      "label": "p1",
       "kind": "Variable",
       "detail": "$CompilationError$",
       "sortText": "B",
-      "insertText": "c3",
+      "insertText": "p1",
       "insertTextFormat": "Snippet"
     },
     {
-      "label": "outputNameString",
+      "label": "p2",
       "kind": "Variable",
-      "detail": "string",
+      "detail": "$CompilationError$",
```

Comment:
@nadeeshaan is this expected? `$CompilationError$`

Reply:
No, this particular issue is an existing one, where the compiler fails to derive the variable's type and then generate it as CompilationError type.